pg_backup_api/requirements.txt

```diff
@@ -1,12 +1,30 @@
-barman >= 2.16
-connexion[swagger-ui] >= 2.6.0; python_version>="3.6"
-# 2.3 is the last version that supports python 3.4-3.5
-connexion[swagger-ui] <= 2.3.0; python_version=="3.5" or python_version=="3.4"
-# connexion requires werkzeug but connexion < 2.4.0 does not install werkzeug
-# we must peg werkzeug versions below to fix connexion
-# https://github.com/zalando/connexion/pull/1044
-werkzeug == 0.16.1; python_version=="3.5" or python_version=="3.4"
-swagger-ui-bundle >= 0.0.2
-python_dateutil >= 2.6.0
-setuptools >= 21.0.0
-Flask == 1.1.2
+argcomplete==2.0.0
+attrs==21.4.0
+barman>=2.17
```

Comment:
Should this be `>=2.16` to match the system package Barman requirement?

Reply:
The package requirement is >2.16, not >= (though that may just be a typo in this comment). I don't have an opinion either way on >2.16 vs >=2.17 to be the consistent choice. This one does appear to be the odd one out, so I'd just go with >2.16 everywhere.

Reply:
You're right - it's `>2.16`. It wasn't so much a typo as me mis-reading it altogether. We probably should make it consistent but on the other hand `>=` is more typical in python so it could just stay how it is.